### ISSUE TITLE
DropDownList: aria-hidden attribute on the animation container and the inner listbox do not allow the Narrator reader to find the aria-activedescendant

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#5366]
### Bug report
DropDownList: aria-hidden attribute on the animation container and the inner listbox do not allow the Narrator reader to find the aria-activedescendant.

Related to: https://github.com/telerik/kendo-ui-core/issues/3380

### Reproduction of the problem
Run the DropDownList demo in Edge and with MS Narrator running: https://demos.telerik.com/kendo-ui/dropdownlist/keyboard-navigation. Notice that when focusing on the DropDownList, the selected value will not be announced. 

### Current behavior
The selected item is not announced.

### Expected/desired behavior
The selected item should be announced

### Environment

* **Kendo UI version:** 2019.3.917
* **Browser:** [Edge]

**Comment by michal-ayash** [comment]
Any plan to fix it?

**Comment by veselints** [comment]
Hi,
We have not yet scheduled the above bug fix for implementation. Based on your question, I have bumped the Priority of this issue, which will move it sooner in our priority queue. Keep tracking this item, so you will be notified as soon as its state changes.

**Comment by martintabakov** [comment]
Also mentioned in #1484401

**Comment by veselints** [comment]
Here are two possible approach that should resolve the `aria-hidden="true"` + `aria-activedescendant` issue present for the widget:

1. Use also an `aria-labelledby` attribute, which would point to the `k-input` element. This way the rendered DropDownList value will be announced upon focusing the widget. That is how the case is approached in Kendo UI for Angular:
https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/

  That is also what the WCAG WAI-ARIA recommendations example implements:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

  The downside of the above would be that when using a reader different from Narrator, it would announce first the `k-input` element value AND the content of the `aria-activedescendant`. Such an issue is not present in Kendo UI for Angular, as the pop-up element of the widget is not present in the DOM when it is not opened. In the WAI-ARIA example, the `aria-activedescendant` attribute is not used at all.

2. Upon focusing the widget, set the `aria-hidden` attribute to `false` and respectively set that back to `true` when blurring the DropDownList. I should test whether the timing of setting the attributes would be appropriate in order to achieve the expected result (announce the selected value).

**Comment by veselints** [comment]
ComboBox also has an `aria-activedecendant` pointing inside its poup.

**Comment by veselints** [comment]
The final solution is to:
* Substitute the `aria-haspopup=true` with `aria-haspopup=listbox`. That is needed to signify that the type of the popup is `listbox`;
* When the popup is closed (not visible) the `aria-activedescendant` points to the `k-input` element instead. This way the selection is always announced upon focusing the widget;

**Comment by michal-ayash** [comment]
Thanks. In which version is it fixed?

**Comment by martintabakov** [comment]
Hi, please try version 2020.3.1118
